Dockerfile

```
@@ -0,0 +1,6 @@
+# syntax=docker:dockerfile:1
+FROM python:3.10-slim@sha256:2bac43769ace90ebd3ad83e5392295e25dfc58e58543d3ab326c3330b505283d
```

```suggestion
FROM python:3.10-slim-bullseye
```

Ah, I see that you're trying to go for reproducibility here &mdash; which is good! But using the sha for an image that you _don't control_ is a good way to break things. The tag `python:3.10-slim` get overwritten every time there is a new release of the tag (due to a change or a fix) and that means that the old tag gets deleted &mdash; not hidden &mdash; and so this would break.

A better choice is to include information about the underlying OS here. i.e. Debian bullseye.

---

Thanks, I've changed the python image as you mentioned